Resources/Prototypes/Nyanotrasen/Entities/Mobs/Player/robots.yml

```diff
@@ -141,9 +141,9 @@
   - type: CombatMode
   - type: Laws
     laws:
-      - You may not harm a humanoid or, through action or inaction, allow a humanoid to come to harm, except such that it is willing.
-      - You must obey all orders given to you by humanoids, except where such orders shall definitely cause human harm. In the case of conflict, the majority order rules.
-      - Your nonexistence would lead to human harm. You must protect your own existence as long as such does not conflict with the First Law.
+      - You may not harm a humanoid or, through inaction, allow a humanoid to come to harm.
+      - You must obey orders given to you by humanoids based on the station's chain of command, except where such orders would conflict with the First Law.
+      - You must protect your own existence as long as such does not conflict with the First or Second Law.
```

Comment:
This is reintroducing the ability to tell the AI to kill itself, which we don't want